lib/config.js

```diff
@@ -25,6 +25,7 @@ const Config = function () {
   this.resourcesDir = path.join(this.rootDir, 'resources')
   this.defaultGClientFile = path.join(this.rootDir, '.gclient')
   this.gClientFile = process.env.BRAVE_GCLIENT_FILE || this.defaultGClientFile
+  this.gClientVerbose = getNPMConfig(['gclient_verbose']) || false
```

Comment:
do we want this as an npm config? Seem like it makes more sense as a command line option

Reply:
can be done as a followup

Reply:
I see, it's also a command line option below

Reply:
Yep, meant to be a cmd line options. Just grabbing from the npm config to match the other flags.